Cargo.toml

```diff
@@ -7,3 +7,8 @@ edition = "2021"
 
 [dependencies]
 chrono = "0.4"
+
+[profile.release]
+lto = true
```

Comment:
I'm trying to be as minimal as possible with this example, keeping the Cargo.toml short.

I'll discuss lto and alternatives to it in more detail in my upcoming book in the section on inlining.